lib/bambora/bank/batch_report_resource.rb

```diff
@@ -83,7 +83,7 @@ def batch_report_body(request_data)
           merchant_id: client.merchant_id,
           pass_code: api_key,
           sub_merchant_id: client.sub_merchant_id,
-        )
+        ).compact
```

Comment:
Oh wow, we can even keep the previous implementation mostly intact with that suggestion 🙏🏻